doc/install/index.rst

```diff
@@ -4,109 +4,69 @@
 Installing Ceph
 ===============
 
-There are various options for installing Ceph. Review the documention for each method before choosing the one that best serves your needs.
+There are several different ways to install Ceph.  Please choose the
+method that best suites your needs.
 
-We recommend the following installation methods:
+.. raw:: html
 
-   * cephadm
-   * Rook
+   <style type="text/css">div.body h3{margin:5px 0px 0px 0px;}</style>
+   <h3>Recommended Methods</h3>
 
+:ref:`Cephadm <cephadm>` installs and manages a Ceph cluster using containers and
+systemd, with tight integration with the CLI and dashboard GUI.
 
-We offer these other methods of installation in addition to the ones we recommend:
+* cephadm supports only Octopus and newer releases.
+* cephadm is fully integrated with the new orchestration API and
+  fully supports the new CLI and dashboard features to manage
+  cluster deployment.
+* cephadm requires container support (podman or docker) and
+  Python 3.
 
-   * ceph-ansible
-   * ceph-deploy (no longer actively maintained)
-   * Deepsea (Salt)
-   * Juju
-   * Manual installation (using packages)
-   * Puppet
+`Rook <https://rook.io/>`_ deploys and manages Ceph clusters running
+in Kubernetes, while also enabling management of storage resources and
+provisioning via Kubernetes APIs.  Rook is the recommended way to run Ceph in
+Kubernetes or to connect an existing Ceph storage cluster to Kubernetes.
 
+* Rook supports only Nautilus and newer releases of Ceph.
+* Rook is the preferred method for running Ceph on Kubernetes, or for
+  connecting a Kubernetes cluster to an existing (external) Ceph
+  cluster.
+* Rook fully suports the new orchestrator API. New management features
+  in the CLI and dashboard are fully supported.
 
-Recommended Methods of Ceph Installation
-========================================
+.. raw:: html
 
-cephadm
--------
+   <h3>Other methods</h3>
```

Comment:
doesn't really make sense. why not using the ReST section style?

Reply:
yeah, probably
```rst
Other methods
~~~~~~~~~~~~~
```
?

Reply:
updated.  i was trying to avoid having these sections appear on the left-side toctree, but having recommended and not recommended is fine at least.

we could do subheadings for each method too, but it's slightly weird when some of them are nested (ceph-deploy, manual) and some jump to other sections.  but probably all should jump to other sections i guess.  the toctree behavior of sphinx seems quite wonky to me.. no way to make the sections within a doc not generate heading items there.  :/

Reply:
Mabe

```rest
.. toctree::
   :titlesonly:
```

or 
```rest
.. toctree::
   :includehidden:
```
would work?